app/data/events.json

```diff
@@ -1,4 +1,11 @@
 [
+	{
+		"date": "2024-08-29T18:00:00.000-04:00",
+		"link": "https://www.eventbrite.com/e/boston-ts-club-v-the-compiler-strikes-back-tickets-???",
```

Comment:
Eventbrite isn't letting me log in 🙃 but as soon as it does I will post back!

Reply:
```suggestion
		"link": "https://www.eventbrite.com/e/boston-ts-club-v-the-compiler-strikes-back-supabase-tickets-962814643517?aff=oddtdtcreator",
```

Reply:
@JoshuaKGoldberg what's the `?aff=oddtdtcreator` part?

Reply:
Oh oops that shouldn't be there

Reply:
```suggestion
		"link": "https://www.eventbrite.com/e/boston-ts-club-v-the-compiler-strikes-back-supabase-tickets-962814643517",
```